apps/services/search-indexer/src/migrate.ts

```diff
@@ -266,26 +278,27 @@ class App {
   }
```

Comment:
Because the migrate has been halted on "createTemplate", I think it would be great catch the call to "putTemplate"

Example:
`return client.indices.putTemplate({
      name: templateName,
      body: config,
    }).catch(reason => logger.info('ES error while createTemplate ', {reason: reason, config: config}))`